kge/job/train.py

```diff
@@ -252,18 +254,21 @@ def load(self, filename: str) -> str:
         Returns job id of the job that created the checkpoint."""
         self.config.log("Loading checkpoint from {}...".format(filename))
         checkpoint = torch.load(filename, map_location="cpu")
+        if self.config.get("job.type") == "train" and checkpoint["type"] == "package":
+            raise ValueError("Can not continue training on a packaged model")
```

Comment:
!= checkpoint

Reply:
currently default type name is "train". Since this is in all old checkpoints, I would just leave it like this.

Reply:
Ok, then != train